client/components/toggle/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { PropTypes } from 'react';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormToggle from 'components/forms/form-toggle/compact';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import { sanitize } from 'dompurify';
+
+const renderToggleText = ( text ) => {
+	return (
+		text ? <span className="toggle__text">{ text }</span> : null
+	);
+};
+
+const renderFieldDescription = ( description ) => {
```

Comment:
same here... this is repeated across several components now, so prob an opportunity to DRY (can be a in a separate PR)